### DISCUSSION TITLE
Outreach FOSS4G 2022 - Pythia Project

### DISCUSSION THREAD

**Discussion by acocac** [ProjectPythia/projectpythia.github.io#273] [Q&A]
Hi Pythia Project core team,

In collaboration with European Pangeo enthusiasts incl. @annefou @tinaok @pl-marasco we're preparing a Pangeo 101 workshop for the FOSS4G 2022 meeting (see some details [here](https://talks.osgeo.org/foss4g-2022-workshops/talk/NF8BKU/)). We'd like to highlight the Project Pythia as the primary educational resource to learn Pangeo open-source stack.

We're wondered if you have a sort of brochure, slide deck or markdown file explaining the main features of the project. We've found very pertinent to point to the workshop attendes all exciting material covered by Pythia foundations and the recent community-driven cookbooks!

Feel free to reply here or also contribute to the markdown file where we aim to describe the Project Pythia (see [here](https://github.com/pangeo-data/foss4g-2022/blob/main/tutorial/afterword/pythia.md)).

**Comment by dcamron** [comment]
We have an open issue (#148) for a contributor handout, but not so much for a general advertisement. We have a few conference slides (that can be provided if helpful!), and of course the main [projectpythia.org](https://projectpythia.org) landing page, but I think there's definitely some room for an overview handout like this!

I'll make sure to add this to this week's meeting agenda,
> [Thursday, 4 August 2022 at 11:00 MDT (GMT-6)](https://www.google.com/calendar/event?eid=MW90a2Q4ZWt2dmdpaWp2amlxdW9qZnNlNWxfMjAyMjA4MDRUMTcwMDAwWiBjXzRxcHZmMzE2YWZkOW12MGNpN2QydWlhZm9nQGc&ctz=GMT-06:00)

for discussion. If any of you are able and would like to join, consider chatting with us; we can also accommodate other meeting times. Either way, we can use this discussion to figure out how best to share the work here. Thanks!

**Comment by clyne** [comment]
Here is a recent [presentation](https://docs.google.com/presentation/d/1js9iR2bmNj7rkJSHU9kvB5037KZnK4mlmWF8Twnggis/edit?usp=sharing) on Pythia that might be useful. 

Thanks for reaching out. We're happy to help if we can.

**Comment by brian-rose** [comment]
This will be great to discuss at this weeks's meeting!

In the mean time, I just want to call attention to the fact that [this About page](https://projectpythia.org/about.html#presentations-about-project-pythia) on the main landing site does have a chronological list of Pythia-related presentations with links to slides etc.

**Comment by acocac** [comment]
Hi @dcamron @clyne @brian-rose thanks for all positive responses to my post. All material of the Pythia project is amazing. The presentation looks great too. We'll check how to display the slides in the FOSS4G Jupyter book, and also point to the About page. We'll tag you in the repository or mention here when the content is ready for a revision.

Finally, Pythia meeting times are ok for me, but I am on leave today. I hope to join to future meetings. I usually refer the Pythia foundations to contributors of the EDS book, a community-driven initiative for highlighting scalable, shareable and reproducible environmental science. I'll open an issue in the EDSbook repository as a reminder to submitting it to the Pythia resource gallery.

**Comment by acocac** [comment]
Following-up on my previous message, here's [the section of the Pythia project](https://pangeo-data.github.io/foss4g-2022/afterword/pythia.html) in the FOSS4G 2022 Pangeo 101 workhop. Please feel free to suggest changes in the section or others you might want to contribute (: We're recognizing all type of contributions in the README 🙏

**Reply by clyne** [reply]
This looks great. Thanks for including. One tiny suggestion would be to include the URL for the main Pythia site somewhere: https://projectpythia.org/

**Reply by acocac** [reply]
I've added to a TODO lost in https://github.com/pangeo-data/foss4g-2022/issues/66. 

Please also feel free to suggest resources of recent Pangeo/Pythia trainings or any other relevant active projects consuming/developing Pangeo stack as we have highlighted for Europe.

**Reply by clyne** [reply]
Terrific. Thanks!
